data/remnant/remnant ships.txt

```diff
@@ -1183,33 +1188,30 @@ ship "Peregrine"
 		licenses
 			"Remnant Capital"
 		"cost" 41627000
-		"shields" 16400
-		"hull" 10600
+		"shields" 15400
+		"hull" 8600
 		"required crew" 9
 		"bunks" 25
 		"mass" 350
 		"drag" 4.3
 		"heat dissipation" 0.85
-		"fuel capacity" 1100
+		"fuel capacity" 900
 		"ramscoop" 3.25
-		"cargo space" 59
+		"cargo space" 49
```

Comment:
```suggestion
		"cargo space" 59
```
59 cargo space is pretty average for a medium warship, this isn't a balance issue.